src/Timestamps/LargeTimestampsHTMLFormatter.php

```diff
@@ -22,10 +22,10 @@ class LargeTimestampsHTMLFormatter implements TimestampsFormatterInterface
 
     private $fmtTime;
 
-    public function __construct()
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.